resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb

```diff
@@ -84,16 +75,13 @@ start() {
     # startproc will change users, so make sure that user has permission
     # to access the present working directory.
     cd "${INSTALL_DIR}"
-    startproc -u "${USER}" -l "${LOGFILE}" -p "${PIDFILE}" "/opt/puppetlabs/server/apps/${realname}/bin/${realname} start >> /var/log/puppetlabs/${realname}/${realname}-daemon.log 2>&1"
+    startproc -u "${USER}" -l "${LOGFILE}" -p "${PIDFILE}" -w -- "${INSTALL_DIR}/bin/${realname}" start >/dev/null 2>&1
+    [ ! -z $(pgrep -f <%= EZBake::Config[:uberjar_name] %>) ]
```

Comment:
that seems reasonable.

i think you may want `-n` here instead of `! -z` for readability and I'd recommend putting the subshell in "" to ensure that a value is here, even if it is the empty string.